nibiru/clients/perp.py

```diff
@@ -36,6 +36,7 @@ def params(self):
         output = MessageToDict(proto_output)["params"]
 
         sdk_dec_fields = [
+            # "maintenanceMarginRatio",
```

Comment:
This is not in the perp query anymore, but moved to the vpool queries. We should remove this commented line